src/main/java/org/verdictdb/metastore/ScrambleMetaStore.java

```diff
@@ -153,15 +153,17 @@ public void addToStore(ScrambleMetaSet scrambleMetaSet) throws VerdictDBExceptio
     }
 
     // insert a new entry
+    String replacedString = "";
     StringBuilder insertSqls = new StringBuilder();
     for (ScrambleMeta meta : scrambleMetaSet) {
       InsertValuesQuery q = createInsertMetaQuery(meta);
       String s = QueryToSql.convert(conn.getSyntax(), q);
       LOG.debug("Adding a new scramble meta entry with the query: {}", s);
       insertSqls.append(s);
       insertSqls.append("; ");
+      replacedString = insertSqls.toString().replace(" 'timestamp  ", " timestamp '");
```

Comment:
There is probably a better way to do this.. I'm prepending `timestamp` to the timestamps column in the query, but becausue of where I'm doing it it gets quoted.